src/ui/widgets/video_capture.py

```
@@ -126,95 +149,96 @@ def _update_frame(self):
+                            line_height = max(line_height, text_height)
+                        label_width = max(right - left, max_text_width + 2 * label_padding)
+                        label_height = (line_height + 6) * len(label_lines) + 2 * label_padding
+                        # Center label above bounding box, keep within frame
+                        label_left = max(left + (right - left)//2 - label_width//2, 0)
+                        label_right = min(label_left + label_width, frame_w)
+                        label_top = max(top - label_height - 10, 0)
+                        label_bottom = top - 10
+                        # Draw filled rectangle
+                        cv2.rectangle(frame, (label_left, label_top), (label_right, label_bottom), box_color, cv2.FILLED, lineType=cv2.LINE_AA)
+                        # Draw border
+                        cv2.rectangle(frame, (label_left, label_top), (label_right, label_bottom), (255,255,255), 1, lineType=cv2.LINE_AA)
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)